src/BuildScriptGenerator/ManifestFilePropertyKeys.cs

```diff
@@ -20,5 +20,7 @@ public static class ManifestFilePropertyKeys
         internal const string PythonVersion = nameof(PythonVersion);
 
         internal const string HugoVersion = nameof(HugoVersion);
+
+        internal const string AzFunctionPlatform = "az-function-platform"; 
```

Comment:
We can just call it `PlatformName` and this need not be specific to azure functions